### PR TITLE
These are the keycloak docs not those of redhat-sso ;-)

### PR DESCRIPTION
**Pull request by codrinbucur**
These are the keycloak docs not those of redhat-sso ;-)

### CODE REVIEW

**Comment by stianst** [comment]
Actually they are for both Keycloak and RH-SSO. The quickstart repo should use a variable and not be hardcoded.

**Comment by stianst** [comment]
@pedroigor this change will break RH-SSO docs, but I pressume Keycloak docs are currently broken. Other guides use a variable to point to the quickstarts repo.

File: authorization_services/topics/hello-world-deploy.adoc [hunk 1]
```diff
@@ -66,7 +66,7 @@ This change specifies to the policy enforcer to redirect users to a `/app-authz-
 To build and deploy the application execute the following command:
 
 ```bash
-$ cd redhat-sso-quickstarts/app-authz-jee-vanilla
+$ cd keycloak-quickstarts/app-authz-jee-vanilla
```
Comment (pedroigor) [inline authorization_services/topics/hello-world-deploy.adoc]: Thanks, @codrinbucur. Could you just change that block to:

```
[source, subs="attributes"]
----
$ cd {quickstartRepo_dir}/app-authz-jee-vanilla
$ mvn clean package wildfly:deploy
----
```

**Review by pedroigor — CHANGES_REQUESTED** [review]

File: authorization_services/topics/hello-world-deploy.adoc (hunk 1 above)
Comment (codrinbucur) [reply authorization_services/topics/hello-world-deploy.adoc]: Done

**Review by pedroigor — APPROVED** [review]

**Review by abstractj — APPROVED** [review]

**Review by stianst — APPROVED** [review]

**Pull request merged**